.travis.yml

```diff
@@ -17,6 +17,7 @@ env:
    - DJANGO="Django<1.9"
    - DJANGO="Django<1.10"
    - DJANGO="Django<1.11"
+   - DJANGO="Django<1.12"
```

Comment:
@jarussi-luizalabs this env install minor versions than `1.12`... it's just to test `1.11.X` versions.